lib/v2/tju/oaa/index.js

```diff
@@ -0,0 +1,114 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const { finishArticleItem } = require('@/utils/wechat-mp');
+const { parseDate } = require('@/utils/parse-date');
+const timezone = require('@/utils/timezone');
+
+const oaa_base_url = 'http://oaa.tju.edu.cn/';
+const repo_url = 'https://github.com/DIYgod/RSSHub/issues';
+
+const pageType = (href) => {
+    if (!href.startsWith('http')) {
+        return 'in-site';
+    }
+    const url = new URL(href);
+    if (url.hostname === 'mp.weixin.qq.com') {
+        return 'wechat-mp';
+    } else if (url.hostname === 'oaa.tju.edu.cn') {
+        return 'tju-oaa';
+    } else {
+        return 'unknown';
+    }
+};
+
+module.exports = async (ctx) => {
+    const type = ctx.params && ctx.params.type;
+    let path, subtitle;
+
+    switch (type) {
+        case 'news':
+            subtitle = '新闻动态';
+            path = 'xwdt.htm';
+            break;
+        case 'notification':
+            subtitle = '通知公告';
+            path = 'tzgg.htm';
+            break;
+        default:
+            subtitle = '新闻动态';
+            path = 'xwdt.htm';
+    }
+    let response = null;
+    try {
+        response = await got({
+            method: 'get',
+            url: oaa_base_url + path,
+            headers: {
+                Referer: oaa_base_url,
+            },
+        });
```

Comment:
got requests can be simplified
```suggestion
        response = await got(oaa_base_url + path, {
            headers: {
                Referer: oaa_base_url,
            },
        });
```